forge-gui/res/cardsfolder/upcoming/kardum_patron_of_flames.txt

```diff
@@ -0,0 +1,19 @@
+Name:Kardum, Patron of Flames
+ManaCost:2 R R
+Types:Legendary Creature Demon
+PT:4/3
+K:Haste
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigCounter | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME attacks, put a flame counter on it, then seek a card with mana value equal to the number of flame counters on it and exile that card face down.
+SVar:TrigCounter:DB$ PutCounter | CounterType$ FLAME | CounterNum$ 1 | ValidCards$ Self | SubAbility$ DBSeek
+SVar:DBSeek:DB$ ChangeZone | Origin$ Library | Destination$ Exile | ExileFaceDown$ True | AtRandom$ True | NoShuffle$ True | NoLooking$ True | NoReveal$ True | ChangeNum$ 1 | ChangeType$ Card.cmcEQX+YouOwn | RememberChanged$ True 
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Battlefield | Destination$ Graveyard | Execute$ TrigReturn | TriggerDescription$ When NICKNAME dies, put the cards exiled with it into their owner's hand. At the beginning of the end step of your next turn, discard those cards.
+SVar:TrigReturn:DB$ ChangeZoneAll | ChangeType$ Card.IsRemembered+ExiledWithSource | RememberChanged$ True | Origin$ Exile | Destination$ Hand | SubAbility$ DBDelay
+SVar:DBDelay:DB$ DelayedTrigger | DelayedTriggerDefinedPlayer$ You | Mode$ Phase | Phase$ End of Turn | Execute$ TrigDiscardExiled | RememberObjects$ Remembered | TriggerDescription$ At the beginning of the end step of your next turn, discard those cards.
```

Comment:
hmm, is this doing the next turn or the current turn?

Reply:
Next one

Reply:
Shouldn't it have `NextTurn$ True` then?

Reply:
Yeah I tried that but that that didn't work. My initial line was like :SVar:DBDelay:DB$ DelayedTrigger | NextTurn$ True | ValidPlayer$ You | Mode$ Phase | Phase$ End of Turn | Execute$ TrigDiscardExiled | RememberObjects$ Remembered | TriggerDescription$ At the beginning of the end step of your next turn, discard those cards.

Reply:
Tried it again with that line. Didn't trigger the discard

Reply:
Meandering Towershell uses the same method, that's how I found it

Reply:
Hmm, Meandering Towershell has `ThisTurn$ True` 🤷‍♂️ 
@tool4ever any insight here?

Reply:
`DelayedTriggerDefinedPlayer` sets it up starting for the next turn
In addition `ThisTurn` will make sure it doesn't work on another turn (e.g. the turn could be ended by Obeka)